app/helpers/schema_helper.py

```diff
@@ -145,13 +134,12 @@ def get_aliases(cls, survey_json):
         for block in cls.get_blocks(survey_json):
             for question in cls.get_questions_for_block(block):
                 for answer in question['answers']:
-                    if 'alias' in answer and answer['alias'] not in aliases:
+                    if 'alias' in answer:
+                        assert answer['alias'] not in aliases, 'Duplicate alias found: ' + answer['alias']
```

Comment:
Not that I know of; I think when we checked we would need to change the EB instance internally to turn it on, there be dragons!